actions/redoc/template.hbs

```diff
@@ -1,3 +1,35 @@
+Skip to content
+Search or jump to…
+Pull requests
+Issues
+Marketplace
+Explore
+ 
+@sudoku-lord 
+lob
+/
+lob-openapi
+Template
+23
+4
+3
+Code
+Issues
+Pull requests
+1
+Actions
+Security
+Insights
+Settings
+lob-openapi/actions/redoc/template.hbs
+@sudoku-lord
+sudoku-lord fixed redoc script & collapser button colour (#221)
+Latest commit a3fdc6c 7 days ago
+ History
+ 3 contributors
+@jho44@sudoku-lord@hilary
+701 lines (677 sloc)  38.8 KB
+  
```

Comment:
what's this? o.o

Reply:
oh man I... I don't know why that got added, but I'm gonna delete it